packages/eslint-config/index.js

```diff
@@ -10,6 +10,16 @@ const hasTypescript = hasConfig([
   { type: 'file', pattern: 'tsconfig.json' },
 ]);
 
+let config;
+
+if (hasReact) {
+  config = {
+    react: {
+      version: 'detect',
+    },
+  };
+}
```

Comment:
@jomaxx this should fix the thing where you must specific a react version.

Reply:
awesome! thanks for doing this